core/CMakeLists.txt

```diff
@@ -24,12 +30,5 @@ if(TTK_BUILD_DOCUMENTATION)
       DESTINATION
         ${CMAKE_INSTALL_DATAROOTDIR}/doc/ttk
         )
-    install(
```

Comment:
why removing this block?
where did the `install` call go

Reply:
This `install` call copies the content of the top-level `doc` folder inside the installation prefix. Since Doxygen now handles directly the `splash.png` picture, only the first `install` call is necessary.

Reply:
makes sense. thanks.